Earthfile

```diff
@@ -100,6 +103,9 @@ build-yawollet-image:
     ARG USEROS
     ARG USERARCH
 
+    ARG MACHINE_FLAVOR=c1.2
+    ARG VOLUME_TYPE=storage_premium_perf6
```

Comment:
do we want to have these in the repo? guess these "sensible defaults" only apply to stackit right?